crates/zizmor/src/main.rs

```diff
@@ -142,6 +144,101 @@ struct App {
     inputs: Vec<String>,
 }
 
+/// Shell with auto-generated completion script available.
+#[derive(Clone, Copy, Debug, Eq, Hash, PartialEq)]
+#[non_exhaustive]
+pub enum Shell {
```

Comment:
This is not a public API so IMO we can make it exhaustive + drop the `pub` qualifier 🙂 

```suggestion
enum Shell {
```

Reply:
done